src/app/QuickStarts/quickstarts/start-a-recording.tsx

```diff
@@ -111,31 +110,32 @@ There are two tabs within the Recordings page:
 
 [Active Recordings]{{highlight active-recordings-tab}} and [Archived Recordings]{{highlight archived-recordings-tab}}.
 
-Active recordings are recordings that only exist only within the target JVM. Archived recordings are recordings that have been saved from the target JVM and copied to Cryostat's storage volume.
+**Active recordings** are recordings that only exist only within the target JVM. **Archived recordings** are recordings that have been saved from the target JVM and copied to [APP]'s storage volume.
 
-We will start a recording while on the Active tab.
+Start an active recording:
 
-1. Click [Create]{{highlight recordings-create-btn}} to go to the Custom Flight Recording Form.
+1. Click [Create]{{highlight recordings-create-btn}} to open the **Custom Flight Recording Form**. 
+[If you have a smaller viewport, the \`Create\` button may not be immediately visible. In this case, you can click on the kebab button (three vertical dots) to reveal additional options, including the \`Create\` button."]{{admonition note}}
```

Comment:
Try:

> [If you have a smaller viewport, the \`Create\` button may not be immediately visible. In this case, you can click on the kebab button (three vertical dots) to reveal additional options, including the \`Create\` ~button.~"

Remove the button word at the end (since we are talking about options).